test/test_mavlink_protocol.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * This file is part of the Camera Streaming Daemon project
+ * This file is part of the Dronecode Camera Manager
```

Comment:
I suggest to add a brief description of what the app does. Sample [here](https://github.com/dronecore/DroneCore/blob/3cd51ed0b356d1eb5cac1d785104ef61ab3699d3/example/fly_qgc_mission/fly_qgc_mission.cpp#L1-L25).